frontend/packages/noobaa-storage-plugin/src/components/object-bucket-page/create-ob.tsx

```diff
@@ -0,0 +1,112 @@
+  const [state, dispatch] = React.useReducer(commonReducer, defaultState);
+
+  React.useEffect(() => {
+    const obj: K8sResourceKind = {
+      apiVersion: `${NooBaaObjectBucketModel.apiGroup}/${NooBaaObjectBucketModel.apiVersion}`,
+      kind: NooBaaObjectBucketModel.kind,
+      metadata: {
+        name: state.name,
+      },
+      spec: {
+        ssl: false,
+      },
+    };
+    if (state.scName) {
+      obj.spec.storageClassName = state.scName;
+    }
+    dispatch({ type: 'setPayload', payload: obj });
+  }, [state.name, state.scName]);
+
+  const save = (e: React.FormEvent<EventTarget>) => {
+    e.preventDefault();
+    dispatch({ type: 'setProgress' });
+    k8sCreate(NooBaaObjectBucketModel, state.payload)
+      .then((resource) => {
+        dispatch({ type: 'unsetProgress' });
+        history.push(resourceObjPath(resource, referenceFor(resource)));
+      })
+      .catch((err) => {
+        dispatch({ type: 'setError', message: err.message });
+        dispatch({ type: 'unsetProgress' });
+      });
+  };
+
+  return (
+    <div className="co-m-pane__body co-m-pane__form">
+      <Helmet>
+        <title>Create Object Bucket</title>
+      </Helmet>
+      <h1 className="co-m-pane__heading co-m-pane__heading--baseline">
+        <div className="co-m-pane__name">Create Object Bucket</div>
+        <div className="co-m-pane__heading-link">
+          <Link to="/k8s/cluster/objectbucket/~new" id="yaml-link" replace>
+            Edit YAML
+          </Link>
+        </div>
+      </h1>
+      <form className="co-m-pane__body-group" onSubmit={save}>
+        <div>
+          <div className="form-group">
+            <label className="control-label co-required" htmlFor="ob-name">
+              Object Bucket Name
+            </label>
+            <div className="form-group">
+              <input
+                className="pf-c-form-control"
+                type="text"
+                onChange={(e) => {
+                  dispatch({ type: 'setName', name: e.currentTarget.value });
+                }}
+                placeholder="my-object-bucket"
+                aria-describedby="ob-name-help"
+                id="ob-name"
+                name="obName"
+                pattern="[a-z0-9](?:[-a-z0-9]*[a-z0-9])?"
+                required
+              />
+              <p className="help-block" id="ob-name-help">
+                If not provided, a generic name will be generated.
+              </p>
+            </div>
+            <div className="form-group">
+              <StorageClassDropdown
+                onChange={(sc) =>
+                  dispatch({ type: 'setStorage', name: _.get(sc, 'metadata.name') })
+                }
+                id="storageclass-dropdown"
+                describedBy="hide_storage_class_default_help"
+                required
+                name="storageClass"
+              />
+              <p className="help-block" id="ob-name-help">
```

Comment:
You have used `ob-name-help` as id at two places.

Reply:
ack